debug/org.eclipse.debug.tests/src/org/eclipse/debug/tests/viewer/model/VisibleVirtualItemValidator.java

```diff
@@ -46,11 +46,12 @@ public int getLength() {
 	@Override
 	public boolean isItemVisible(VirtualItem item) {
 		int position = 0;
-		while (item.getParent() != null) {
-			position += item.getIndex().intValue();
-			item = item.getParent();
+		VirtualItem currentItem = item;
```

Comment:
```suggestion
		VirtualItem rootItem = item;
```

Reply:
It is a variable that reflects the current item during tree traversal. It will have the role of the root item after traversal has completed but calling it `rootItem` during traversal would not be correct in my opinion.

Reply:
You're right, I read `currentItem.getParent()` and automatically thought of trees and depth.